### PR TITLE
New Game - Spylite

### PR DESCRIPTION
**Pull request by hylianux**
tl;dr - Spylite is a new game, hoping to be placed under the Microlite20 category due to its basis in ruleset.

I want to make sure it's clear that this is a different game entirely -- this is not a reskin of the Microlite20 character sheet.  

This game is a lighter version of Spycraft created using the ruleset of Microlite20.  Differences with Microlite20 include secret agent styled classes, absence of spells (this is a more modern-style RPG), absence of money (replaced by gear points the player can use for equipment supplied by their organization), absence of shields, modified handling of armor, and automatically-calculated skills (I'm not automating any leveling... that's just the way the skills are calculated).  The similarities, however, are evident in the skill+stat roll buttons, as well as the attacks.

Since this game's rules are extended from Microlite20's rules, I'd like this sheet to be displayed under the Microlite20 category if possible.  Let me know if you have any questions :)

### CODE REVIEW

**Comment by Rhallinan** [comment]
The sheet looks good. I'll add it under the Microlite category after I do some testing.
